plotly/figure_factory/_facet_grid.py

```diff
@@ -25,6 +25,7 @@
 THRES_FOR_FLIPPED_FACET_TITLES = 10
 GRID_WIDTH = 1
 
+VALID_TRACE_TYPES = ['scatter', 'scattergl', 'histogram']
```

Comment:
technically other types too though, right? I think that `box`, `histogram2d`, and `bar` might work now